guides/source/action_controller_overview.md

```diff
@@ -1073,10 +1073,10 @@ class ClientsController < ApplicationController
 end
 ```
 
-For this example to work, you have to add the PDF MIME type to Rails. This can be done by adding the following line to the file `config/initializers/mime_types.rb`:
+This example works because [Rails already registers the `"application/pdf"` MIME type](https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/http/mime_types.rb) at startup (alongside other common types). If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example:
 
 ```ruby
-Mime::Type.register "application/pdf", :pdf
+Mime::Type.register(<mime_type>, <symbol>)
```

Comment:
It seems like the linter expects this to be valid ruby.

Reply:
Perhaps the example could be removed altogether, since there's already a link to the method.

I couldn't come up with an example to use, but also didn't want to give the impression that it's super expected practice to add new MIMEs, because Rails supports so many out of the box. I've been tripped by the PDF example many times 😅 and all my apps have a `mime` file that's probably not needed at all.